packages/astro/src/runtime/server/index.ts

```diff
@@ -372,9 +372,11 @@ export async function renderPage(result: SSRResult, Component: AstroComponentFac
   const template = await renderToString(result, Component, props, children);
   const styles = Array.from(result.styles)
     .filter(uniqueElements)
+    .reverse()
```

Comment:
Tiny suggestion: could your PR explanation live here as a comment? Not blocking, and non-essential though. Good fix!